app/gulam_chor/page.tsx

```diff
@@ -43,7 +43,7 @@ export default function Game() {
           />
         </div>
 
-        <Button onClick={createGame} className="w-full">
+        <Button onClick={createGame} className="w-full" disabled={roomId !== ""}>
```

Comment:
We don't need to check the roomId is empty or not
```suggestion
        <Button onClick={createGame} className="w-full" disabled={!roomId}>
```